packages/vue-dot/src/patterns/FormField/fields/ChoiceSliderField.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+	<VSlider
+		v-if="field.items"
+		color="accent"
+		:value="getIndex(field.value)"
+		class="mt-5"
+		v-bind="field.metadata"
+		:thumb-label="thumbLabel"
+		:tick-labels="thickLabels"
+		:max="field.items.length -1"
+		@change="valueChanged"
+	>
+		<template
+			v-if="thumbLabel"
+			#thumb-label="props"
```

Comment:
```suggestion
			#thumb-label="{ value }"
```
C'est toi qui m'avait rappelé qu'on pouvait faire ça :stuck_out_tongue: